docs/src/main/paradox/couchbase.md

```diff
@@ -20,164 +30,117 @@ The table below shows direct dependencies of this module and the second tab show
 
 @@dependencies { projectId="couchbase" }
 
+# Overview
 
-#Usage
+Alpakka Couchbase offers both Akka Streams APIs and a more direct API to access Couchbase:
 
-We will need an @scaladoc[ActorSystem](akka.actor.ActorSystem) and an @scaladoc[ActorMaterializer](akka.stream.ActorMaterializer).
+* `CouchbaseSession` offers a direct API for one-off operations
+* `CouchbaseSessionRegistry` is a Akka extension to keep track and share `CouchbaseSession`s within an `ActorSystem`
+* `CouchbaseSource`, `CouchbaseFlow`, and `CouchbaseSink` offer factory methods to create Akka Streams
```

Comment:
Add links to API docs?

Reply:
Also we need a word for collective definition of Akka Stream `Source`, `Flow` and `Sink`. Akka Stream docs [suggest "operator"](https://doc.akka.io/docs/akka/current/stream/stream-flows-and-basics.html#defining-and-running-streams).

How about

> ... offer factory methods to create Akka Stream operators